docs/source/main_classes/logging.mdx

```diff
@@ -33,14 +33,37 @@ TRANSFORMERS_VERBOSITY=error ./myprogram.py
 ```
 
 Additionally, some `warnings` can be disabled by setting the environment variable
-`TRANSFORMERS_NO_ADVISORY_WARNINGS` to a true value, like *1*. This will disable any warning that is logged using
+`TRANSFORMERS_NO_ADVISORY_WARNINGS` to a true value, like _1_. This will disable any warning that is logged using
```

Comment:
```suggestion
`TRANSFORMERS_NO_ADVISORY_WARNINGS` to a true value, like *1*. This will disable any warning that is logged using
```

I'm pretty sure we are using `*X*` for italics, so I'm not sure why you're proposing to change this.

@sgugger, should we document that for consistency?

Reply:
Both work in MarkDown, but it's true we use `*` more consistently (it's coming from conversion scripts)

Reply:
@sgugger @stas00 nice spot, I fear that is my vscode autosave feature